### ISSUE TITLE
ActivityIndicator Bug on First Load

### ISSUE THREAD

**Issue by kceb** [callstack/react-native-paper#2332]
### Current behaviour

I'm using ActivityIndicator on a page and every first load I see a momentary glitch where there are two indicators. Video below.

### Expected behaviour

I don't expect a double ActivityIndicator

### Code sample

<ActivityIndicator animating />

### Screenshots (if applicable)

https://d.pr/MJK9rn

### What have you tried

Using other components and seeing if they exhibit same behavior. (They don't)

I thought it might be a metro server thing, but upon generating a release apk and sideloading, I see the same thing. I'm not using any other animation library that would conflict.

I tried react-native's default ActivityIndicator and it doesn't exhibit this behavior.

### Your Environment

| software              | version
| --------------------- | -------
| ios or android        | Android 5.1 API 22
| react-native          | "0.63.3"
| react-native-paper    | 4.2.0
| node                  | v14.8.0
| npm or yarn           | yarn 1.22.4
| expo sdk              | N/A
| react-native-vector-icons | 7.1.0

**Comment by RichardLindhout** [comment]
It only happens with that specific Android version or also on iOS and newer Android versions?

**Comment by bradser** [comment]
FWIW, I saw a similar issue, turns out it was a <Suspense> around an <ActivityIndicator> in my code.

**Comment by CDBridger** [comment]
I see this issue to when I navigate between screens using react-navigation. Would this issue be able to be re opened.

**Comment by fr0staman** [comment]
I see similar issue with react-native-reanimated and react-navigation.

```jsx

import { ActivityIndicator } from "react-native-paper";
import Animated, { FadeOutUp } from "react-native-reanimated";

...

return (
  <Animated.View exiting={FadeOutUp}>
    <ActivityIndicator size={"large"} animating={animating} />
  </Animated.View>
)
```

**Comment by aderyan18** [comment]
How to fixed? I stuck in here case

**Comment by fidelusp** [comment]
FYI I am also facing the issue

**Comment by DenisDov** [comment]
In my case, it's related to how the event loop works. I will need to execute `await new Promise((resolve) => requestAnimationFrame(resolve));`  right before hiding the `ActivityIndicator`.

**Comment by obayit** [comment]
Happened to me after I upgraded from expo 54 to 56. obviously upgraded other expo dependencies too.
But it is consistent and not only on first load.

It got fixed after I upgraded paper from `^5.14.5` to `^5.15.3`.
